jamf2snipe

```diff
@@ -790,7 +790,11 @@ for jamf_type in jamf_types:
                     try:
                         jamf_asset_tag = jamf['{}'.format(tag_split[0])]['{}'.format(tag_split[1])]
                     except:
-                        raise SystemError('No such attribute {} in the jamf payload. Please check your settings.conf file'.format(tag_split))
+                        if jamf_type == 'mobile_devices':
+                            jamf_asset_tag = 'jamfid-m-{}'.format(jamf['general']['id'])
+                        elif jamf_type == 'computers':
+                            jamf_asset_tag = 'jamfid-{}'.format(jamf['general']['id'])
+                        #raise SystemError('No such attribute {} in the jamf payload. Please check your settings.conf file'.format(tag_split))
```

Comment:
Instead of raising a system Error, or commenting it out, maybe we can log an error and continue (skip the asset) under a final else statement?

Reply:
maybe something like
```
else:
    logging.error("Could not generate an asset tag for this device. Skipping")
    # Dump the object for debugging. 
    logging.verbose(jamf)
    continue
```

Reply:
Makes sense, I added that in.